jobclient/java/src/main/java/com/twosigma/cook/jobclient/Job.java

```diff
@@ -1401,7 +1401,7 @@ public static Job parseFromJSON(JSONObject json) throws JSONException {
         List<Job> jobs = new ArrayList<Job>(jsonArray.length());
         for (int i = 0; i < jsonArray.length(); ++i) {
             JSONObject json = jsonArray.getJSONObject(i);
-            jobs.add(parseFromJSON(json));
+            jobs.add(parseFromJSON(json, decorator));
```

Comment:
Is there a reasonable unit test we could write for this?

Reply:
I gotta come clean, I have no idea what the `decorator` even did in this code. Do you have any good test in mind?